docs/src/pages/components/autocomplete/Tags.tsx

```diff
@@ -57,6 +57,16 @@ export default function Tags() {
           />
         )}
       />
+      <Autocomplete
+        multiple
+        id="tags-readOnly"
+        options={top100Films.map((option) => option.title)}
+        defaultValue={[top100Films[12].title, top100Films[13].title]}
+        readOnly
+        renderInput={(params) => (
+          <TextField {...params} label="readOnly" placeholder="Favorites" />
+        )}
+      />
```

Comment:
Please don't overload demos, this demo is not about readonly, it has nothing to do here

```suggestion
```

Reply:
It shows readOnly functionality for multiple options. We have `freeSolo` and `filterSelectedOptions` functionality there as well.

Reply:
What I mean is that multiple demos would improve the DX.